test/integration/fixtures/markdown-in.txt

```diff
@@ -1,6 +1,6 @@
 # This is a heading with *italic*
```

Comment:
I know this is unrelated, but it's weird that this file is `.txt`. Just figured I'd point that out.

Reply:
Why is it weird? :)

Reply:
It's a Markdown fixture file–I'd just expect the file extension to be either `.md` or `.markdown`, which are the usual extensions for a file written in Markdown. That way editors will syntax highlight it, etc.

Reply:
Oh, this was to differentiate between HTML (.html) and plain text (.txt) input.